org.freedownload.Manager.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: org.freedownload.Manager
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: .
+command: fdm
+tags:
+  - proprietary
+finish-args:
+  - --socket=wayland 
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-music
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-pictures
+modules:
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-libraries=headers
+      - ./b2 install variant=release link=shared runtime-link=shared cxxflags="$CXXFLAGS" linkflags="$LDFLAGS" -j $FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.bz2
+        sha256: 475d589d51a7f8b3ba2ba4eda022b170e562ca3b760ee922c146b6c65856ef39
+        
+  - name: libtorrent
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/arvidn/libtorrent/releases/download/v2.0.6/libtorrent-rasterbar-2.0.6.tar.gz
+        sha256: 438e29272ff41ccc68ec7530f1b98d639f6d01ec8bf680766336ae202a065722
+  
+  - name: fdm
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib
+      - install -Dm 755 apply_extra.sh /app/bin/apply_extra
+      - install -Dm 644 -t /app/share/metainfo org.freedownload.Manager.metainfo.xml
+      - install -Dm 644 -t /app/share/applications org.freedownload.Manager.desktop
+      - install -Dm 644 -t /app/share/icons/hicolor/256x256/apps org.freedownload.Manager.png
+      - install -Dm 755 fdm.sh /app/bin/fdm 
+      - install -d /app/lib/ffmpeg
+      # For extra-data to be able to extract the .deb
+      - install -Dm 755 /{usr,app}/bin/ar
+      - cp /usr/lib/$(gcc --print-multiarch)/libbfd-*.so /app/lib
+
+    sources:
+      - type: file
+        path: apply_extra.sh
+      
+      - type: file
+        path: fdm.sh
+
+      - type: file
+        path: org.freedownload.Manager.desktop
+      
+      - type: file
+        path: org.freedownload.Manager.png
+      
+      - type: file
+        path: org.freedownload.Manager.metainfo.xml
+
+      - type: extra-data
+        url: https://dn3.freedownloadmanager.org/6/latest/freedownloadmanager.deb
```

Comment:
since this is x86_64 only then you should instruact flathub to only build x86_64

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
The last line contains the declaration for x86_64 only, I have no idea why the bot is building for aarch

Reply:
because this just mean "ignore this source". I linked to the documentation.